packages/plugin-table/assets/tableUI.css

```diff
@@ -1,5 +1,6 @@
 table.table-picker {
     position: absolute;
+    z-index: 1;
```

Comment:
I'm surprised that works. Did you make sure it does?

Reply:
What do you expect this to do? If you don't understand why it's supposed to work, you can read the PR explanation and the link that goes with it. I tested in Chrome and Firefox on my machine, if you think I should try somewhere else, feel free to tell me.